packages/data-context/src/DataContext.ts

```diff
@@ -429,7 +429,11 @@ export class DataContext {
   }
 
   async destroy () {
-    const destroy = util.promisify(this.coreData.servers.gqlServer?.destroy || (() => {}))
```

Comment:
```
util.promisify(() => {})
```
This noop didn't work because it never resolved. We replaced it with `() => Promise.resolve()`